runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -211,6 +211,355 @@ doubleMaxMinHelper(TR::Node *node, TR::CodeGenerator *cg, bool isMaxOp)
+   TR::Register* fromIndexReg = cg->gprClobberEvaluate(node->getChild(firstCallArgIdx+4));
+
+   // Registers
+   TR::Register* resultReg          = cg->allocateRegister();
+   TR::Register* maxIndexReg        = cg->allocateRegister();
+   TR::Register* s1VecStartIndexReg = fromIndexReg;
+   TR::Register* s2VecStartIndexReg = cg->allocateRegister();
+   TR::Register* loadLenReg         = cg->allocateRegister();
+   TR::Register* char1IndexReg      = resultReg;
+
+   TR::Register* s1PartialVReg      = cg->allocateRegister(TR_VRF);
+   TR::Register* s2PartialVReg      = cg->allocateRegister(TR_VRF);
+   TR::Register* s2Char1RepVReg     = cg->allocateRegister(TR_VRF);
+   TR::Register* tmpVReg            = cg->allocateRegister(TR_VRF);
+
+   // register dependencies
+   TR::RegisterDependencyConditions * regDeps = new (cg->trHeapMemory()) TR::RegisterDependencyConditions(0, 13, cg);
+   regDeps->addPostCondition(s1ValueReg  , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(s1LenReg    , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(s2ValueReg  , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(s2LenReg    , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(fromIndexReg, TR::RealRegister::AssignAny);
+
+   regDeps->addPostCondition(resultReg         , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(maxIndexReg       , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(s2VecStartIndexReg, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(loadLenReg        , TR::RealRegister::AssignAny);
+
+   regDeps->addPostCondition(s1PartialVReg , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(s2PartialVReg , TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(s2Char1RepVReg, TR::RealRegister::AssignAny);
+   regDeps->addPostCondition(tmpVReg       , TR::RealRegister::AssignAny);
+
+   // Labels
+   TR::LabelSymbol* labelStart               = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelFindS2Head          = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelLoadLen16           = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelLoadLenDone         = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelExtractFirstCharPos = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelMatchS2Loop         = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelMatchS2LoopSetup    = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelS2PartialMatch      = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelLoadResult          = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelResultDone          = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelStringNotFound      = generateLabelSymbol(cg);
+   TR::LabelSymbol* labelDone                = generateLabelSymbol(cg);
+
+   // Start
+   TR::Instruction* cursor = generateS390LabelInstruction(cg, TR::InstOpCode::LABEL, node, labelStart);
+   cursor->setStartInternalControlFlow();
```

Comment:
See https://github.com/eclipse/omr/pull/3106. We're deprecating these APIs and only setting ICF on labels. We need to change these accordingly.

Reply:
OK. Changed to `labelStart->setStartInternalControlFlow()`.